src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java

```diff
@@ -34,7 +34,7 @@ protected void collectParameters(Consumer<Parameter<?>> parameterCollector) {
 
     @Override
     protected void collectAcceptedFileExtensions(Consumer<String> fileExtensionConsumer) {
-        fileExtensionConsumer.accept("xml");
+        fileExtensionConsumer.accept("zip");
```

Comment:
do we always enforce compression or should we accept both?

Reply:
currently we do always enforce compression.
even when there is ".xml" in the targetpath.

Reply:
Currently we seem to do so, would be cool to support both

Reply:
I like forcing it, and so avoiding GB-big files "poluting" our storage.